packages/webapp/src/routes/deploy/+page.svelte

```diff
@@ -1,24 +1,17 @@
 <script lang="ts">
-	import { PageHeader, StrategyShortTile } from '@rainlanguage/ui-components';
-	import { Button, Input, Toggle } from 'flowbite-svelte';
+	import { PageHeader, StrategyShortTile, InputRegistryUrl } from '@rainlanguage/ui-components';
+	import { Toggle } from 'flowbite-svelte';
 	import { page } from '$app/stores';
 
 	const { strategyDetails } = $page.data;
 
-	let newRegistryUrl = '';
-	let advancedMode = false;
-
-	const loadRegistryUrl = () => {
-		// add the registry url to the url params
-		window.history.pushState({}, '', window.location.pathname + '?registry=' + newRegistryUrl);
-		// reload the page
-		window.location.reload();
-	};
+	let newRegistryUrl = localStorage.getItem('registry') || '';
```

Comment:
can we move this logic into InputRegistryUrl instead of spreading it across two components?